app/models/info_request.rb

```diff
@@ -331,13 +331,15 @@ def update_url_title
     unique_url_title = url_title
     suffix_num = 2 # as there's already one without numeric suffix
     conditions = id ? ["id <> ?", id] : []
-    while InfoRequest.
+
+    while !InfoRequest.
       where(:url_title => unique_url_title).
         where(conditions).
-          first do
+          pluck(:id).empty? do
```

Comment:
It looks like it could be quicker to use `#any?` here:

```ruby
logger = Logger.new(STDOUT)
logger.formatter = ActiveSupport::Logger::SimpleFormatter.new
ActiveRecord::Base.logger = logger

url_title = InfoRequestBatch.first.info_requests.first.url_title
# => "content_filtering_software_229"

InfoRequest.where(url_title: url_title).where([]).pluck(:id).empty?
#   (9.6ms)  SELECT "info_requests"."id" FROM "info_requests"  WHERE "info_requests"."url_title" = 'content_filtering_software_229'
# => false

InfoRequest.where(url_title: url_title).where([]).any?
#   (0.9ms)  SELECT COUNT(*) FROM "info_requests"  WHERE "info_requests"."url_title" = 'content_filtering_software_229'
# => true
```

```diff
diff --git a/app/models/info_request.rb b/app/models/info_request.rb
index 0580cc8..cbfc1a1 100644
--- a/app/models/info_request.rb
+++ b/app/models/info_request.rb
@@ -332,10 +332,10 @@ class InfoRequest < ActiveRecord::Base
     suffix_num = 2 # as there's already one without numeric suffix
     conditions = id ? ["id <> ?", id] : []

-    while !InfoRequest.
+    while InfoRequest.
       where(:url_title => unique_url_title).
         where(conditions).
-          pluck(:id).empty? do
+          any? do
```